supporting_scripts/playwright/setupUsers.sh

```
@@ -0,0 +1,18 @@
+
+# We use the supporting scripts to create users
+
+set -e
+
+cd ..
+
+if [ ! -d "venv" ]; then
+    python -m venv venv
+fi
+
+source venv/bin/activate
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'source' in place of '.' is undefined.

(SC3046)